Resources/Prototypes/StatusIcon/faction.yml

```
@@ -136,4 +136,17 @@
   icon:
     sprite: /Textures/Interface/Misc/job_icons.rsi
     state: Syndicate
+
+- type: factionIcon
+  id: BloodCultFaction
+  isShaded: true
+  priority: 11
+  showTo:
+    components:
+      - ShowAntagIcons
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)